imports/plugins/core/accounts/client/containers/editGroupContainer.js

```diff
@@ -0,0 +1,206 @@
+import { Meteor } from "meteor/meteor";
+import _ from "lodash";
+import React, { Component } from "react";
+import classnames from "classnames";
+import PropTypes from "prop-types";
+import { Packages } from "/lib/collections";
+import { Reaction } from "/client/api";
+import { composeWithTracker } from "/lib/api/compose";
+import { List, ListItem, Card, CardHeader, CardBody, Alerts } from "@reactioncommerce/reaction-ui";
+import PermissionsList from "../components/permissionsList";
+import GroupForm from "../components/groupForm";
+import { groupPermissions } from "../helpers/accountsHelper";
+
+class EditGroupContainer extends Component {
+  static propTypes = {
+    accounts: PropTypes.array,
+    groups: PropTypes.array,
+    onChangeGroup: PropTypes.func,
+    packages: PropTypes.array,
+    selectedGroup: PropTypes.object
+  };
+
+  constructor(props) {
+    super(props);
+    const { accounts, selectedGroup, groups } = props;
+
+    this.state = {
+      alertArray: [],
+      selectedGroup: selectedGroup || {},
+      isEditing: false,
+      groups,
+      accounts
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { groups, selectedGroup } = nextProps;
+    this.setState({ groups, selectedGroup });
+  }
+
+  selectGroup = grp => {
```

Comment:
done. we probably should add that to our eslintrc, so that it gets flagged for everyone too when working